server/routes/polls/polls.js

```diff
@@ -0,0 +1,86 @@
+const express = require('express');
+const router = express.Router();
+const authentication = require('../../middleware/authentication');
+const PollsMongoModel = require('../../models/Polls');
+const { cloudinary } = require('../../cloudinary/cloudinary');
+
+router.post('/create', authentication, async (req, res) => {
+  try {
+    console.log('this is user', req.user);
+    const userId = req.user.id;
+    const { question, friend, imagesData } = req.body;
+    const uploadedResponseToCloudinaryForFirstImage = await cloudinary.uploader.upload(imagesData[0], {
+      upload_preset: 'team_pumpkin',
+    });
+    const uploadedResponseToCloudinaryForSecondImage = await cloudinary.uploader.upload(imagesData[1], {
+      upload_preset: 'team_pumpkin',
+    });
+
+    const newUserPollData = {
+      userId,
+      url1: uploadedResponseToCloudinaryForFirstImage.url,
+      url2: uploadedResponseToCloudinaryForSecondImage.url,
+      friend,
+      question,
+    };
+    const newUserPollDataSaveToMongo = new PollsMongoModel(newUserPollData);
+    newUserPollDataSaveToMongo.save().then((response) => {
+      res.status(200).json(response);
+    });
+  } catch (err) {
+    const error = {
+      msg: err,
+    };
+    console.log(err);
+    res.status(400).json(error);
+  }
+});
+
+router.put('/update', authentication, async (req, res) => {
```

Comment:
This can become /polls/:id, or just be a put request to /polls if we pass in the ID in the body
```suggestion
router.put('/polls/:id', authentication, async (req, res) => {
```
```suggestion
router.put('/polls', authentication, async (req, res) => {
```